tests/driver_test.go

```diff
@@ -42,28 +50,50 @@ var _ = Describe("Network Driver", func() {
 		})
 
 		Context("on AllocateNetwork", func() {
-			// Dunno. in standard windows drivers, it throws NotImplementedError
+			It("responds with empty AllocateNetworkResponse, nil", func() {
+				req := network.AllocateNetworkRequest{}
+				resp, err := contrail_driver.AllocateNetwork(&req)
+				Expect(resp).To(Equal(&network.AllocateNetworkResponse{}))
+				Expect(err).ShouldNot(HaveOccurred())
+			})
```

Comment:
All these change request are about non-existing features, they will be added in another pr.